graviti/operation/frame.py

```diff
@@ -164,16 +231,18 @@ def do(  # pylint: disable=invalid-name, unused-argument
         )
 
 
-class UpdateData(DataFrameOperation):
+class UpdateData(DataModification):
     """This class defines the operation that updates the data of a DataFrame.
 
     Arguments:
         data: The data for updating.
 
     """
 
+    _file_arrays: List["FileSeries"]
+
     def __init__(self, data: "DataFrame") -> None:
-        self.data = data
+        self._data = data
```

Comment:
The init methods in AddData and UpdateData are the same. Move it to DataModification?